scripts/create_config.py

```diff
@@ -21,8 +21,23 @@
 # Access the lists using the argument names
 gmsl_pulsename = args.gmsl_pulse
 gmst_pulsename = args.gmst_pulse
-pulse_years = list(map(int, args.pulse_years))
-gases = args.gases
+
+if args.pulse_years:
+    pulse_years = list(map(int, args.pulse_years))
+else:
+    print("No pulse years specified")
+    print("Defaulting to 2020 pulse year")
```

Comment:
I like this default year and gas solution. Please make a note in the README that these 2 args are not required and what their defaults are.